database/sql/V028__index_settlements_tx_hash.sql

```diff
@@ -0,0 +1,2 @@
+-- When getting orders by tx hash we index the settlements table by tx_hash.
+CREATE INDEX settlements_tx_hash ON settlements USING HASH (tx_hash);
```

Comment:
Why use a hash index instead of a btree index?

Reply:
The difference is that a BTREE index facilitates ordering of the indexed data while a HASH doesn't. We don't need any ordering so I picked HASH which I assume is faster (because otherwise why does it exist?).

Reply:
> The difference is that a BTREE index facilitates ordering

Makes perfect sense, we only need random access here. I was mostly curious because I remember reading somewhere that the default indexing method was `BTREE` and didn't ever think to try and optimize indices for the specific kind of queries that are needed.